osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -451,5 +434,65 @@ private void load(OsuColour colour)
                 valueText.Colour = colour.GrayB;
             }
         }
+
+        private class MetadataSegment : Container
+        {
+            private readonly OsuSpriteText header;
+            private readonly FillFlowContainer<OsuSpriteText> content;
+
+            private const int fade_time = 250;
+
+            public string HeaderText
+            {
+                set
+                {
+                    header.Text = value;
+                }
+            }
+
+            public string ContentText
+            {
+                set
+                {
+                    if (value == "")
+                        FadeOut(fade_time);
+                    else
+                    {
+                        FadeIn(fade_time);
+                        content.Children = value.Split(' ').Select(text => new OsuSpriteText

```

Comment:
word wrap still needs to be implemented at a framework level, so not sure how i feel about local solutions like this (i'd rather stuff just wrapped badly until it's fixed).

also tags will probably be formatted differently in the future (individual tags will have a box/hover effect) so i think i preferred your previous method of splitting for tags only.